app/views/stream/_about.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container about">
+  <section class="page-header">
+    <h1>Activity streams</h1>
+  </section>
+
+  <section class="main">
+    <article class="article-block">
+      <div class="article-contents">
+        <h4 class="header">How it works</h4>
+        <p> Activity Streams show you how other people approach to the problems. </p>
+        <p> When you have submitted a solution to at least one exercise, then the stream starts getting filled with the solutions that other people have submitted to the same exercise.</p>
+        <p> It will differentiate your submissions from the others by <a href="http://exercism.io/languages">langauge track</a> </p>
+        <p> You will be able to view and comment on others' submissions <p>
+      </div>
+    </article>
+    <hr>
+    <article class="article-block">
+      <div class="article-contents">
+        <h4 class="header">Track Mentor</h4>
+        <p>Want to be a Track Mentor? Working on exercises and giving great feedback is the best way to get noticed!</p>
```

Comment:
Instead of suggesting that people be a "track mentor" explicitly here, let's answer the question about what they can/should do when they're looking at other people's code.

E.g.

* Ask questions about how something works.
* Point out places where the code is confusing, complex, or hard to understand.
* Ask questions about the trade-offs that people made, or alternate solutions that they considered.
* Suggest that they look at something from the standard library that might help simplify their solution.

And perhaps, also, a sentence about the value of _pointing out what they observe_ more than _suggesting specific changes_, since that leaves the door open to more exploration.

Or something to that effect.

I'd love to hear from @jtigger, @Insti and @kotp who have all given a lot of feedback on the site.